src/pages/Cafeteria/PCCafeteriaPage/components/PCMenuBlocks/index.tsx

```diff
@@ -1,80 +1,29 @@
-import { convertDateToSimpleString } from 'utils/ts/cafeteria';
+import useDinings from 'pages/Cafeteria/hooks/useDinings';
+import MealDetail from 'pages/Cafeteria/PCCafeteriaPage/components/MealDetail';
+import PCMealImage from 'pages/Cafeteria/PCCafeteriaPage/components/PCMealImage';
+import { MEAL_TYPE_MAP, PLACE_ORDER } from 'static/cafeteria';
 import styles from './PCMenuBlocks.module.scss';
 
-interface MealDetailProps {
-  item: CafeteriaMenu;
-  setMealDetail: (element: JSX.Element) => void;
-}
-
-function MealDetail({ item, setMealDetail }: MealDetailProps): JSX.Element {
-  const handleCloseModal = () => {
-    setMealDetail(<div />);
-  };
-
-  return (
-    <button
-      className={styles.overlay}
-      onClick={handleCloseModal}
-      type="button"
-    >
-      <button
-        className={styles.modal}
-        onClick={(e) => e.stopPropagation()}
-        type="button"
-      >
-        <div className={styles['modal-header']}>
-          <div className={styles['modal-header__place-chip']}>
-            <div className={styles['modal-header__place']}>{item.place}</div>
-            {item.soldout_at && <span className={`${styles['modal-header__chip']} ${styles['modal-header__chip--sold-out']}`}>품절</span>}
-            {!item.soldout_at && item.changed_at && <span className={`${styles['modal-header__chip']} ${styles['modal-header__chip--changed']}`}>변경됨</span>}
-          </div>
-          <div className={styles['modal-header__detail']}>
-            {!!item.kcal && `${item.kcal}kcal`}
-            {!!item.kcal && !!item.price_card && !!item.price_cash && '•'}
-            {!!item.price_card && !!item.price_cash && `${item.price_card}원/${item.price_cash}원`}
-          </div>
-          <button
-            type="button"
-            aria-label="닫기"
-            className={styles['modal-header__close']}
-            onClick={handleCloseModal}
-          >
-            <CloseIcon />
-          </button>
-        </div>
-        <img
-          className={styles.modal__image}
-          src={item.image_url!}
-          alt="menu"
-        />
-      </button>
-    </button>
-  );
-}
-
 interface Props {
   mealType: MealType;
-  recentDate: boolean;
+  isRecent: boolean;
 }
 
-export default function PCMenuBlocks({ mealType, recentDate }: Props) {
+export default function PCMenuBlocks({ mealType, isRecent }: Props) {
   const { currentDate } = useDatePicker();
-  const { cafeteriaList } = useCafeteriaList(convertDateToSimpleString(currentDate));
+  const { dinings } = useDinings(currentDate);
 
-  const filteredCafeteriaList = cafeteriaList.filter((item) => item.type === mealType);
-  const sortedCafeteriaList = filteredCafeteriaList.sort((a, b) => {
-    const indexA = placeOrder.indexOf(a.place);
-    const indexB = placeOrder.indexOf(b.place);
+  const filteredDinings = dinings
+    .filter((meal) => meal.type === mealType)
+    .filter((meal) => !meal.menu.some((dish) => dish.name.includes('미운영')));
```

Comment:
이 로직에 문제가 없다면 여기도 이렇게 사용하는게 더 간단해보여요!
```suggestion
  const filteredDinings = dinings
    .filter((meal) => meal.type === mealType && !meal.menu.some((dish) => dish.name.includes('미운영')));
```